srcpkgs/fontmanager/template

```diff
@@ -13,4 +13,4 @@ maintainer="Antonio Malcolm <antonio@antoniomalcolm.com>"
 license="GPL-3.0-or-later"
 homepage="https://fontmanager.github.io/"
 distfiles="https://github.com/FontManager/master/archive/${version}.tar.gz"
```

Comment:
Please update distfiles to use https://github.com/FontManager/font-manager

Reply:
Done.